packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -122,10 +131,11 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
          * NB: the operatorKit is responsible for
          *
          * @param {CctpTxEvidence} evidence
+         * @param {RiskAssessment} riskAssessment
          * @param {string} operatorId
          */
-        attest(evidence, operatorId) {
-          const { operators, pending } = this.state;
+        attest(evidence, riskAssessment, operatorId) {
+          const { operators, pending, risks } = this.state;
           trace('submitEvidence', operatorId, evidence);
```

Comment:
while you're here, I had missed updating this string
```suggestion
          trace('attest', operatorId, evidence);
```

Reply:
Done